content/client-server-api/modules/voip_events.md

```diff
@@ -171,18 +171,34 @@ In response to an incoming invite, a client may do one of several things:
 
 ##### Streams
```

Comment:
can we add something to https://pr1602--matrix-spec-previews.netlify.app/client-server-api/#common-fields about the new field?